src/measure_theory/measure/outer_measure.lean

```diff
@@ -1114,7 +1114,7 @@ begin
 end
 
 lemma induced_outer_measure_exists_set {s : set α}
-  (hs : induced_outer_measure m P0 m0 s < ∞) {ε : ℝ≥0} (hε : 0 < ε) :
+  (hs : induced_outer_measure m P0 m0 s ≠ ∞) {ε : ℝ≥0} (hε : 0 < ε) :
```

Comment:
While you're at it, could you also change `hε` to `ε ≠ 0`?

Reply:
The proper way to do it is to change `ennreal.lt_add_right`. I'm going to do this but let's merge this part first.